registry/static/css/award_document.css

```diff
@@ -0,0 +1,105 @@
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    background-color: grey; /* #FAFAFA */
+    font-family: "Arial","Helvetica Neue","Helvetica","sans-serif";
+    font-size: 12pt;
+}
+
+main > * {
+    margin: 40px
+}
+
+h1 {
+    font-size: 16pt;
+    text-align: center;
+}
+
+ul {
+    list-style-type: none;
+}
+
+aside ul {
+    text-align: center;
+    margin-bottom: 50px;
+}
+
+footer {
+    margin: 160px 40px 0 40px;
+    font-style: italic;
+    font-size: 11pt;
+}
+
+main p {
+    font-style: italic;
+    text-align: justify;
+}
+
+p:nth-of-type(3) {
+    margin-top: 50px;
+    text-align: center;
```

Comment:
> zarovnáno na střed, ale moc se mi to nelíbí

I zarovnaná i nezarovnaná varianta vypadají dobře, takže proč ne. To je asi čistě věc vkusu. :smile:

Reply:
Po diskuzi na callu jsme došli k tomu, že zarovnaná varianta „vypadá profesionálněji“. :smile: